<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librepcb::editor::BomGeneratorDialog</class>
 <widget class="QDialog" name="librepcb::editor::BomGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate BOM</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1">
   <item row="0" column="1">
    <widget class="QComboBox" name="cbxBoard"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Attributes:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Board:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblAssemblyVariant">
     <property name="text">
      <string>Assembly Variant:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Output File:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="cbxAssemblyVariant"/>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="edtAttributes">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>Comma-separated list of custom attributes (optional)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="edtOutputPath"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnBrowseOutputDir">
       <property name="text">
        <string>Browse Output Directory</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblNote">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Gray rows are not exported (do not mount).</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbxBoard</tabstop>
  <tabstop>edtAttributes</tabstop>
  <tabstop>edtOutputPath</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>btnBrowseOutputDir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
